### ISSUE TITLE
Spawn error on windows

### ISSUE THREAD

**Issue by ReneMueller4P** [NativeScript/NativeScript#10532]
### Issue Description

Hi, when i run nx nativescript-dasboard:android i get errors on Windows.

### Relevant log output (if applicable)

```shell
C:\Entwicklung\Frontend git:[master] npm run serve:android --verbose npm verb cli C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js npm info using npm@10.5.0 npm info using node@v18.20.2 npm verb title npm run serve:android npm verb argv "run" "serve:android" "--loglevel" "verbose" npm verb logfile logs-max:10 dir:C:\Users\334\AppData\Local\npm-cache_logs\2024-04-29T06_12_29_888Z- npm verb logfile C:\Users\334\AppData\Local\npm-cache_logs\2024-04-29T06_12_29_888Z-debug-0.log  
frontend@0.0.0 serve:android gulp app && set IS_PROD=0 && nx run nativescript-dashboard:android  
[08:12:30] Using gulpfile C:\Entwicklung\Frontend\gulpfile.js [08:12:30] Starting 'app'... [08:12:30] Finished 'app' after 114 ms  
nx run nativescript-dashboard:android  
^[[18;1R^C^C ―――――――――――――――――――――――― 🤖 Running NativeScript CLI within apps/nativescript-dashboard  ns debug android --force  <hr></hr> Error: spawn EINVAL at ChildProcess.spawn (node:internal/child_process:414:11) at Object.spawn (node:child_process:761:9) at runCommand (C:\Entwicklung\Frontend\node_modules@nativescript\nx\src\utils\executors.js:222:44) at checkOptions (C:\Entwicklung\Frontend\node_modules@nativescript\nx\src\utils\executors.js:270:21) at C:\Entwicklung\Frontend\node_modules@nativescript\nx\src\utils\executors.js:370:21 at new Promise (<anonymous>) at commonExecutor (C:\Entwicklung\Frontend\node_modules@nativescript\nx\src\utils\executors.js:10:12) at C:\Entwicklung\Frontend\node_modules@nativescript\nx\src\executors\build\executor.js:7:43 at Generator.next (<anonymous>) at C:\Entwicklung\Frontend\node_modules\tslib\tslib.js:169:75 { errno: -4071, code: 'EINVAL', syscall: 'spawn' }</anonymous></anonymous>  NX spawn EINVAL
```

### Environment

```yaml
OS: Windows 11 10.0.22631
CPU: (20) x64 13th Gen Intel(R) Core(TM) i7-1370P
Shell: Not Found
node: 20.12.2
npm: 10.3.0
nativescript: Not Found

# android
java: Not Found
ndk: Not Found
apis: 30, 32, 33, 34
build_tools: 33.0.0, 33.0.2, 34.0.0
system_images:
  - android-31 | Google APIs Intel x86_64 Atom
  - android-33 | Google APIs Intel x86_64 Atom
  - android-34 | Google APIs Intel x86_64 Atom

# ios
xcode: Not Found
cocoapods: Not Found
python: Not Found
python3: Not Found
ruby: Not Found
platforms: Not Found
```

### Dependencies

```json
"dependencies": {
  "@angular/animations": "17.3.3",
  "@angular/cdk": "17.3.3",
  "@angular/cdk-experimental": "17.3.3",
  "@angular/common": "17.3.3",
  "@angular/compiler": "17.3.3",
  "@angular/core": "17.3.3",
  "@angular/forms": "17.3.3",
  "@angular/localize": "17.3.3",
  "@angular/platform-browser": "17.3.3",
  "@angular/platform-browser-dynamic": "17.3.3",
  "@angular/pwa": "17.3.3",
  "@angular/router": "17.3.3",
  "@auth0/angular-jwt": "^5.1.0",
  "@azure/msal-angular": "^3.0.5",
  "@azure/msal-browser": "^3.2.0",
  "@bradmartin/nativescript-urlhandler": "^2.0.1",
  "@ckeditor/ckeditor5-build-classic": "^40.2.0",
  "@ckeditor/ckeditor5-enter": "^40.0.0",
  "@ckeditor/ckeditor5-watchdog": "^40.0.0",
  "@frontend/xplat-scss": "file:libs/xplat/scss/src",
  "@nativescript-community/fonticon": "^3.0.0",
  "@nativescript-community/ui-pager": "14.1.8",
  "@nativescript/angular": "^17.0.0",
  "@nativescript/background-http": "6.0.0",
  "@nativescript/camera": "^5.1.1",
  "@nativescript/core": "~8.7.2",
  "@nativescript/firebase-auth": "^3.2.4",
  "@nativescript/imagepicker": "^3.0.0",
  "@nativescript/mlkit-barcode-scanning": "^2.1.0",
  "@nativescript/mlkit-core": "^2.1.0",
  "@ngneat/until-destroy": "^8.1.4",
  "@ngx-translate/core": "~15.0.0",
  "@ngx-translate/http-loader": "~8.0.0",
  "@nstudio/nativescript-barcodescanner": "^5.0.1",
  "@nstudio/nativescript-cardview": "^2.0.1",
  "@sergeymell/nativescript-svg": "^1.1.2",
  "@triniwiz/nativescript-image-zoom": "^4.2.0",
  "@types/chartist": "^1.0.0",
  "@types/mime": "^3.0.1",
  "@web/styles": "file:libs/xplat/web/scss/src",
  "chartist": "^1.3.0",
  "convert-svg-to-png": "^0.6.4",
  "core-js": "^3.6.5",
  "date-fns": "^2.29.3",
  "dayjs": "^1.10.7",
  "deepmerge-ts": "^5.1.0",
  "document-register-element": "^1.14.10",
  "dotenv-webpack": "^7.1.0",
  "env-cmd": "^10.1.0",
  "frontend": "file:",
  "get-root-path": "^2.0.2",
  "git-repo-info": "^2.1.1",
  "gulp": "^4.0.2",
  "gulp-streamify": "^1.0.2",
  "gulp-svgmin": "^4.1.0",
  "hex-to-css-filter": "^5.4.0",
  "jest-html-reporter": "^3.9.0",
  "jest-util": "^29.7.0",
  "js-scanner-detection": "^1.1.0",
  "jsvat": "^2.5.3",
  "libphonenumber-js": "^1.10.6",
  "mime": "^3.0.0",
  "moment": "^2.29.4",
  "nativescript-barcodescanner": "^4.1.2",
  "nativescript-couchbase-plugin": "^0.9.6",
  "nativescript-dna-deviceinfo": "^3.7.3",
  "nativescript-fonticon": "^8.0.2",
  "nativescript-theme-core": "^2.0.24",
  "nativescript-ui-listview": "15.2.3",
  "ng-chartist": "^8.1.3",
  "ng-process-env": "^1.0.6",
  "ngx-auto-unsubscribe-decorator": "^1.1.0",
  "ngx-extended-pdf-viewer": "^18.0.2",
  "ngx-mask": "^16.3.9",
  "ngx-sse-client": "^3.0.0",
  "ngx-translate-multi-http-loader": "^16.0.2",
  "qrcode-svg": "^1.1.0",
  "require.js": "^1.0.0",
  "rxjs": "^7.5.0",
  "s": "^1.0.0",
  "simple-keyboard": "^3.6.1",
  "simple-keyboard-layouts": "^3.2.2",
  "stacktrace-js": "^2.0.2",
  "tslib": "^2.6.2",
  "util": "^0.12.4",
  "vscode-html-languageservice": "^5.0.6",
  "webpack-bundle-analyzer": "^4.6.1",
  "zone.js": "0.14.2"
},
"devDependencies": {
  "@angular-devkit/architect": "0.1703.3",
  "@angular-devkit/build-angular": "17.3.3",
  "@angular-devkit/core": "17.3.3",
  "@angular-devkit/schematics": "17.3.3",
  "@angular-eslint/eslint-plugin": "17.3.0",
  "@angular-eslint/eslint-plugin-template": "17.3.0",
  "@angular-eslint/template-parser": "17.3.0",
  "@angular-extensions/pretty-html-log": "^5.0.2",
  "@angular/cli": "~17.3.3",
  "@angular/compiler-cli": "17.3.3",
  "@angular/language-service": "17.3.3",
  "@angular/service-worker": "17.3.3",
  "@babel/preset-env": "^7.14.8",
  "@babel/preset-typescript": "^7.16.5",
  "@ckeditor/ckeditor5-angular": "^7.0.0",
  "@faker-js/faker": "^8.4.1",
  "@jest/globals": "^29.7.0",
  "@nativescript/eslint-plugin": "~0.0.4",
  "@nativescript/nx": "^18.0.0",
  "@nativescript/types": "^8.7.0",
  "@nativescript/webpack": "^5.0.21",
  "@ngneat/spectator": "^18.0.1",
  "@ngtools/webpack": "^17.0.7",
  "@nstudio/angular": "^17.0.1",
  "@nstudio/nativescript": "^17.0.1",
  "@nstudio/nativescript-angular": "17.0.1",
  "@nstudio/web": "17.0.1",
  "@nstudio/web-angular": "17.0.1",
  "@nstudio/xplat": "17.0.1",
  "@nx/angular": "18.2.3",
  "@nx/cypress": "18.2.3",
  "@nx/eslint": "18.2.3",
  "@nx/eslint-plugin": "18.2.3",
  "@nx/jest": "18.2.3",
  "@nx/js": "18.2.3",
  "@nx/web": "18.2.3",
  "@nx/workspace": "18.2.3",
  "@schematics/angular": "17.3.3",
  "@testing-library/angular": "^15.2.0",
  "@types/jest": "^29.4.0",
  "@types/node": "^18.16.9",
  "@typescript-eslint/eslint-plugin": "^7.6.0",
  "@typescript-eslint/parser": "^7.6.0",
  "codelyzer": "^0.0.28",
  "cypress": "^13.6.6",
  "dotenv": "^10.0.0",
  "eslint": "8.57.0",
  "eslint-config-prettier": "^9.0.0",
  "eslint-plugin-cypress": "2.13.4",
  "eslint-plugin-prettier": "^5.1.0",
  "gulp-svg-sprite": "^2.0.1",
  "identity-obj-proxy": "^3.0.0",
  "jest": "^29.7.0",
  "jest-canvas-mock": "^2.5.2",
  "jest-css-modules-transform": "^4.4.2",
  "jest-environment-jsdom": "^29.7.0",
  "jest-extended": "^4.0.2",
  "jest-html-reporters": "^3.1.4",
  "jest-jasmine2": "^29.7.0",
  "jest-preset-angular": "14.0.3",
  "jest-preview": "^0.3.1",
  "jest-silent-reporter": "^0.5.0",
  "jest-teamcity": "^1.11.0",
  "jest-transform-css": "^6.0.1",
  "ng-mocks": "^14.11.0",
  "nx": "18.2.3",
  "nx-cloud": "18.0.0",
  "postcss-import": "^15.1.0",
  "prettier": "3.1.1",
  "sass": "^1.56.2",
  "ts-jest": "^29.1.1",
  "ts-loader": "~9.4.2",
  "ts-node": "10.9.2",
  "ts-node-dev": "^2.0.0",
  "typescript": "5.4.4"
}
```

### Please accept these terms

- [X] I have searched the [existing issues](https://github.com/NativeScript/NativeScript/issues) as well as [StackOverflow](https://stackoverflow.com/questions/tagged/nativescript) and this has not been posted before
- [X] This is a bug report
- [X] I agree to follow this project's [Code of Conduct](https://github.com/NativeScript/NativeScript/blob/master/tools/notes/CONTRIBUTING.md#coc)

**Comment by ReneMueller4P** [comment]
The problem is that the command ns.cmd is blocked since node update  [69ffc6d](https://github.com/nodejs/node/commit/69ffc6d50dbd9d7d0257f5b9b403026e1aa205ee)  instead you have to insert shell:true in the spawn options.

**Comment by kdevcse** [comment]
Will this be fixed soon? I started a new app setup with the latest node/npm/NativeScript and this is preventing me from developing 😭

**Comment by rigor789** [comment]
Install the latest NativeScript CLI, this should be fixed in `nativescript@8.7.1`:

```bash
npm i -g nativescript@latest
```
